hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -875,6 +875,12 @@ private FlinkOptions() {
           .defaultValue(HoodieSyncTableStrategy.ALL.name())
           .withDescription("Hive table synchronization strategy. Available option: RO, RT, ALL.");
 
+  public static final ConfigOption<Boolean> HIVE_SYNC_CREATE_MANAGED_TABLE = ConfigOptions
+          .key("hive_sync.create_managed_table")
+          .booleanType()
+          .defaultValue(true)
```

Comment:
Need to make the default value consistent with HiveSyncConfigHolder.HIVE_CREATE_MANAGED_TABLE.

Reply:
Thanks for the reminder, the default has been modified